tests/ui/target-feature/wasm-relaxed-simd.rs

```diff
@@ -0,0 +1,10 @@
+//@ only-wasm32
```

Comment:
I think we only test `wasm32-wasip1` in CI, can we use that target here instead? 

https://github.com/rust-lang/rust/blob/4bc39f028d14c24b04dd17dc425432c6ec354536/src/ci/docker/host-x86_64/test-various/Dockerfile#L62C1-L70C15

Reply:
Done.

However I should note that there are other tests that use `@ only-wasm32` as well.
https://github.com/rust-lang/rust/blob/1def498e3b9adc1ccce3ec5628e9f8755ec96808/tests/ui/target-feature/wasm-safe.rs#L1